Assets/Code/Scripts/LevelManagement/GameManager.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    /*
+     * Basic GameManager implementation where the last checkpoint coordinate is stored.
+     * The istance of a GameManager is handled, even though it seems that Unity does this automatically.
+     */
+    
+    private static GameManager instance;
+    public Vector3 lastCheckPointPos;
+
+    void Awake()
+    {
+        if (instance == null)
+        {
+            instance = this;
+            DontDestroyOnLoad(instance);
+        }
+        else
+        {
+            Destroy(gameObject);
+        }
+    }
```

Comment:
If I am not mistaken (part 2): the idea is that the "developer" needs to manually set the `lastCheckPointPos` of this script from Unity, right? I think it would be cooler to read it from the `spawnPlatform`, which is there for that purpose!

This will reduce the manual operations for each new scene and it will be less error prone.  You can simply add here the following line: 

`lastCheckPointPos = GameObject.FindGameObjectWithTag("SpawnPlatform").transform.position;`

PS: In my opinion, a better approach would be to make  `lastCheckPointPos` a private member, and then put two public methods set and get. But it is not so relevant at this stage, and it could be done in another PR else this will become endless to merge :)